src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -282,14 +281,14 @@ function WorkspaceCategoriesPage({policy}: WorkspaceCategoriesPageProps) {
                         />
                         {isSmallScreenWidth && <View style={[styles.pl5, styles.pr5]}>{getHeaderButtons()}</View>}
                         <View style={[styles.ph5, styles.pb5, styles.pt3]}>
-                            {policy?.connections ? (
+                            {Object.keys(policy?.connections ?? {}).length ? (
                                 <Text>
                                     <Text style={[styles.textNormal, styles.colorMuted]}>{`${translate('workspace.categories.importedFromAccountingSoftware')} `}</Text>
                                     <TextLink
                                         style={[styles.textNormal, styles.link]}
                                         href={`${environmentURL}/${ROUTES.POLICY_ACCOUNTING.getRoute(policyId)}`}
                                     >
-                                        {`${translate('workspace.accounting.qbo')} settings`}
+                                        {`${translate('workspace.accounting.qbo')}, ${translate('common.settings')}`}
```

Comment:
```suggestion
                                        {`${translate('workspace.accounting.qbo')} ${translate('common.settings')}`}
```
- Remove comma
- The word settings is capitalized, should we fix that?

Reply:
fixed: https://github.com/Expensify/App/pull/40496/commits/22468f5551b5ccb53a47279910faeae97ad6401f

Reply:
@hayata-suenaga Remove the comma too